plugins/device/cmd/example/README.md

```diff
@@ -22,3 +38,84 @@ The valid configuration options are:
 * `dir` (`string`: `"."`): The directory to scan for files that will represent fake devices.
 * `list_period` (`string`: `"5s"`): The interval to scan the directory for changes.
 * `unhealthy_perm` (`string`: `"-rwxrwxrwx"`): The file mode permission that if set on a detected file will casue the device to be considered unhealthy.
+
+# Usage
+
+Create two instances of the device, one unhealthy:
+
+```shell
+mkdir -p /tmp/nomad-device
+cd /tmp/nomad-device
+touch device01 && chmod 0777 device01
+touch device02
+```
+
+It should be fingerprinted by the client agent after the ~`list_period`,
```

Comment:
```suggestion
It should be fingerprinted by the client agent after the `list_period`,
```

Reply:
I often use a tilde to indicate "approximately" but I suppose that's not obvious :heavy_check_mark: